### PR TITLE
Changed Maintainer to Committer or Contributor

### PR DESCRIPTION
**Pull request by marksandford**
I tried (though may not have consistently succeeded) to use these guidelines:
'Committer' reserved specifically for cases where having write access to the repos is required for the context. 
'Contributor' used more broadly for anyone contributing to the project, which may include committers as well as others.
'Developer' was often used in the build docs.  Since we're assuming some people may want to work on the builds for their institution without contributing them back, I left this term as-is.

### CODE REVIEW

File: docs/05_contributor_docs/contributor_build_guide.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+* Rename the `docker-compose.yml` file to `docker-compose.bak`
+
+### Step 3: Rename the `docker-compose-build.yml` File
+
+* Rename the `docker-compose-build.yml` file to `docker-compose.yml`
+
+### Step 3a: Alternative for Non-ISLE Committers only (e.g. developers)
+
+For **non-ISLE Committers only e.g. Developers (only)**  you'll need to change the following lines in the newly renamed `docker-compose.yml` file.
```
Comment (bseeger) [inline docs/05_contributor_docs/contributor_build_guide.md]: I would be less specific here - not calling out who as that doesn't really matter, if say, a committer is testing their own stuff.  Something like "If you're working in your own dockerhub environment (ie, not the offical `islandoracollabgroup` environment, then you'll need to change..."

File: docs/05_contributor_docs/contributor_build_guide.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    image: yourdockerimagerepo_orgname/isle-fedora:latest
+    image: yourdockerimagerepo_orgname/isle-apache:latest
+    image: yourdockerimagerepo_orgname/isle-solr:latest
+    image: yourdockerimagerepo_orgname/isle-proxy:latest
+
+```
+
+Change only the images you need to rebuild or customize.
+
+### Step 4: Build the Images in this Suggested Order
```
Comment (bseeger) [inline docs/05_contributor_docs/contributor_build_guide.md]: I'm a little confused here. The docker compose yml file doesn't build images so much as pull them down from docker hub.

File: docs/05_contributor_docs/contributor_build_guide.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+* `docker push islandoracollabgroup/isle-apache:latest`
+* `docker push islandoracollabgroup/isle-fedora:latest`
+* `docker push islandoracollabgroup/isle-mysql:latest`
+* `docker push islandoracollabgroup/isle-proxy:latest`
+* `docker push islandoracollabgroup/isle-solr:latest`
+
+Repeat the process with any additional tags. **Please note:** _Depending on your Internet connection's upload speed this process may take several minutes._
+
+* `docker push islandoracollabgroup/isle-apache:2.4`
```
Comment (bseeger) [inline docs/05_contributor_docs/contributor_build_guide.md]: These now follow release versions - like 1.1 and 1.1.1 - not separate ones like the numbers listed here.  That way you know exactly which ones to pull for a release.

File: docs/05_contributor_docs/contributor_build_guide.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
+<!--- PAGE_TITLE --->
+
+_Expectations:  It can take an up to an average of **2 -4 hours** to read this documentation and complete this installation. Highly dependent on the Developer's local laptop and Internet speed._
```
Comment (bseeger) [inline docs/05_contributor_docs/contributor_build_guide.md]: Is the goal of this document to build a release?  or for a developer to customize their stuff?  It sounds like a mix of both and the release has a few more elements (like building the ubuntu-basebox and tomcat as well).   With the pushing to `islandoracollabgroup` example, that sounds like a release.

Comment (bseeger) [reply docs/05_contributor_docs/contributor_build_guide.md]: Ideally, we'd have one document for each scenario - keep it completely separate so there's no confusion or accidental non-release pushes to the islandoracollabgroup part of dockerhub.  Thoughts?

Comment (dwk2) [reply docs/05_contributor_docs/contributor_build_guide.md]: Your questions are good and valid, but know that this old doc file is semi-obsolete, and is from the earliest days of our docs. Some of these old files are stubs, some will be removed. Hopefully, this weekend.

File: docs/05_contributor_docs/contributor_build_guide.md (hunk 1 above)
Comment (dwk2) [reply docs/05_contributor_docs/contributor_build_guide.md]: You are correct, and I'm going to resolve this conversation as this doc is offline atm, and needs a thorough scrub before, and if, it ever goes live.

**Pull request merged**